src/main/java/com/jmb/springfactory/model/entity/Comment.java

```diff
@@ -0,0 +1,32 @@
+package com.jmb.springfactory.model.entity;
+
+import java.util.Date;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.ManyToOne;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class Comment extends BaseEntity {
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  private User author;
+
+  private String text;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "text" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)